iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/AdjustableChecksVisitor.java

```diff
@@ -79,5 +88,37 @@ private SecondaryLocation adaptSecondaryLocation(SecondaryLocation secondaryLoca
       var shiftedTextRange = locationShifter.computeShiftedLocation(currentCtx, secondaryLocation.textRange);
       return new SecondaryLocation(shiftedTextRange, secondaryLocation.message);
     }
+
+    TextRange toLocationInValuesFile(List<String> valuePaths, InputFile valuesFile) throws IOException {
+      // TODO: get "values.yaml" through context
+//      InputFile valuesFile = currentCtx.getAdditionalFile("values.yaml");
```

Comment:
IMO, the `HelmProcessor` should set what files were used in template evaluation and with one `values.yaml` were used. We shouldn't here implement a similar logic to `HelmFilesystemUtils.additionalFilesOfHelmProjectDirectory`.

Think about upcoming features:
* Support for .helmignore
* Level 4 - using different values.yaml